### PR TITLE
CVE-2022-40626 

### PR DESCRIPTION
**Pull request by zabbixsecurity**
Assigning CNA has been changed for CVE-2022-40626

### CODE REVIEW

File: 2022/40xxx/CVE-2022-40626.json [hunk 1]
```diff
@@ -3,16 +3,118 @@
     "data_format": "MITRE",
```
Comment (cve-team) [inline 2022/40xxx/CVE-2022-40626.json:3]: 'Zabbix' does not match '^([a-zA-Z0-9_\\-\\.]+)@([a-zA-Z0-9_\\-\\.]+)\\.([a-zA-Z]{2,5})$'

**Review by cve-team — CHANGES_REQUESTED** [review]
Auto validation errors:

File: 2022/40xxx/CVE-2022-40626.json (hunk 1 above)
Comment (zabbixsecurity) [reply 2022/40xxx/CVE-2022-40626.json:3]: hi guys,
I have tried multiple times different values for "ASSIGNER" like "security@zabbix.com", "cve@mitre.org" and "Zabbix". Could you please comment on what should be added to this parameter?

Comment (cve-team) [inline 2022/40xxx/CVE-2022-40626.json:3]: 'Zabbix' does not match '^([a-zA-Z0-9_\\-\\.]+)@([a-zA-Z0-9_\\-\\.]+)\\.([a-zA-Z]{2,5})$'

**Review by cve-team — CHANGES_REQUESTED** [review]
Auto validation errors:

File: 2022/40xxx/CVE-2022-40626.json (hunk 1 above)
Comment (anthnysingleton) [reply 2022/40xxx/CVE-2022-40626.json:3]: Hello @zabbixsecurity,

We transferred custodianship of this record from MITRE CNA-LR to Zabbix CNA. In order to get this unblocked please change the assigner value from "Zabbix" to "security@zabbix.com".

Thanks,
Anthony
CVE Content Team

**Pull request closed without merging**